content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -91,73 +103,89 @@ For example:
      "type": "MODIFIED",
      "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "11020", ...}, ...}
    }
-   ...
+   …
    ```
 
-A given Kubernetes server will only preserve a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default.  When the requested watch operations fail because the historical version of that resource is not available, clients must handle the case by recognizing the status code `410 Gone`, clearing their local cache, performing a list operation, and starting the watch from the `resourceVersion` returned by that new list operation. Most client libraries offer some form of standard tool for this logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
+   {{< note >}}
+   Although the API server responded with a `Content-Type` header indicating a JSON response,
+   the response to a _watch_ request is in fact a stream of multiple documents. For JSON format, the client can
+   detect the end of each JSON document and begin parting the next one.
```

Comment:
This extra note is difficult to parse.

Reply:
Let's find _some_ way to make sure the difference in format is clear.

Reply:
nit?: s/parting/parsing/
I think you are indicating that the example response represents a partial response.
The `watch` response is composed of multiple JSON documents.

Reply:
As I understand it, the watch response is invalid JSON that will break some parsers. I think this is relevant to readers, especially anyone implemented a client. Otherwise, a reader might otherwise expect they can use a validating parser and have it just work.

Reply:
I've omitted this change from the latest revision.